ui/elements/Calendar/MonthlyCalendar/MonthlyCalendarGridItem.tsx

```diff
@@ -65,9 +65,17 @@ function MonthlyCalendarGridItem<T>(props: MonthlyCalendarGridItemProps<T>) {
         >
             <DateCircle date={cellDate} />
             {eventsToRender.map((event) => (
-                <React.Fragment key={event?.id}>
+                <div
+                    key={event?.id}
+                    className="truncate max-w-full overflow-hidden whitespace-nowrap text-xs font-medium cursor-pointer"
+                    style={{
+                        lineHeight: '1.2',
+                        position: 'relative',
+                        zIndex: 10,
+                    }}
+                >
```

Comment:
prefer to create a new styled-component for styling here instead of using className and inline styling